extensions/panache/panache-common/runtime/src/main/java/io/quarkus/panache/common/Sort.java

```diff
@@ -100,6 +129,20 @@ public static Sort by(String column, Direction direction) {
         return new Sort().and(column, direction);
     }
 
+    /**
+     * Sort by the given column, in the given order and with null handling.
+     *
+     * @param column the column to sort on, in the given order.
+     * @param direction the direction to sort on
+     * @param nullHandling the null handling to use
+     * @return a new Sort instance which sorts on the given column in the given order.
+     * @see #by(String)
+     * @see #by(String...)
+     */
+    public static Sort by(String column, Direction direction, NullHandling nullHandling) {
```

Comment:
Maybe also add `by(String, NullHandling)`

Reply:
I'm a bit against about this change as internally, we would need to use this constructor:
```java
public Column(String name, Direction direction, NullPrecedence nullPrecedence) {
```
So, we need a direction. We could use the default direction ASCENDING tho.
wdyt?

Reply:
Yes, use the default.

Reply:
Done